app/controllers/projects_controller.rb

```diff
@@ -185,6 +186,18 @@ def pattern
     end
   end
 
+  # Display list of user's projects, sorted by date.
+  def has_member
```

Comment:
>Showing 1 of 1 total issue
Rename has_member to member?. NEW
  def has_member
Severity: MinorFound in [app/controllers/projects_controller.rb](https://codeclimate.com/github/MushroomObserver/mushroom-observer/app/controllers/projects_controller.rb?from_sha=70f70bb7&to_sha=00e724a9) by [rubocop](https://docs.codeclimate.com/docs/rubocop)

If you're looking for a method name that doesn't offend Rubocop, maybe something like:
`enrolled_in`
`roster_includes`
`membership_includes`
`enrollments`
`rostered`
`joined`

Reply:
Lol!  Jason and I talked about this and I settled on has_member as my favorite option.  I'll figure out something.